plugins/export-workbook/README.md

```diff
@@ -1,9 +1,61 @@
-# @flatfile/plugin-export-workbook
+<!-- START_INFOCARD -->
 
-This package exports data in Flatfile a Workbook. Currently, `.xlsx` is supported.
+The `@flatfile/plugin-export-workbook` plugin exports data in a Flatfile Workbook.
 
-`npm i @flatfile/plugin-export-workbook`
+**Event Type:**
+`listener.on('job:ready')` 
 
-## Get Started
+**Supported file types:**
+`.xlsx`
+
+<!-- END_INFOCARD -->
+
+
+> When embedding Flatfile, this plugin should be deployed in a server-side listener. [Learn more](https://flatfile.com/docs/orchestration/listeners#listener-types)
+
+
+## Parameters
```

Comment:
There are a few params not listed here:

`jobName` `string` `optional` The name of job that the plugin will fire on. If not provided, `workbook:downloadWorkbook` will be used.
`excludedSheets` `string[]` `optional` An array of sheets to be excluded from the export
`excludeFields` `string[]` `optional` An array of fields to be excluded from the export
`recordFilter` `Flatfile.Filter` `optional` Allows filtering exported records to `valid` or `error`. By default all records will be exported
`includeRecordIds` `boolean` `optional` Includes the record's ID
```

Reply:
added, thanks!